pkg/clusteragent/autoscaling/workload/model/pod_autoscaler.go

```diff
@@ -34,6 +36,9 @@ const (
 	// statusRetainedActions is the number of horizontal actions kept in status
 	statusRetainedActions = 10
 
+	// statusRetainedRecommendations is the maximum number of horizontal recommendations kept in status
+	statusRetainedRecommendations = 20
```

Comment:
Might be a bit low, it would mean we cover only 10 minutes, we should probably set that to 60 to accomodate 30 mins